app/src/main/java/com/asfoundation/wallet/restore/password/RestoreWalletPasswordFragment.kt

```diff
@@ -8,41 +8,28 @@ import android.text.TextWatcher
 import android.view.LayoutInflater
 import android.view.View
 import android.view.ViewGroup
+import android.view.inputmethod.InputMethodManager
 import com.asf.wallet.R
-import com.asfoundation.wallet.billing.analytics.WalletsEventSender
+import com.asfoundation.wallet.restore.RestoreWalletActivityView
 import com.asfoundation.wallet.ui.iab.FiatValue
 import com.asfoundation.wallet.util.CurrencyFormatUtils
 import com.asfoundation.wallet.util.RestoreErrorType
 import com.jakewharton.rxbinding2.view.RxView
 import dagger.android.support.DaggerFragment
 import io.reactivex.Observable
-import io.reactivex.android.schedulers.AndroidSchedulers
-import io.reactivex.disposables.CompositeDisposable
-import io.reactivex.schedulers.Schedulers
-import kotlinx.android.synthetic.main.restore_wallet_password_layout.*
+import kotlinx.android.synthetic.main.fragment_restore_wallet_password.*
 import kotlinx.android.synthetic.main.wallet_outlined_card.*
 import javax.inject.Inject
 
 class RestoreWalletPasswordFragment : DaggerFragment(), RestoreWalletPasswordView {
 
   @Inject
-  lateinit var restoreWalletPasswordInteractor: RestoreWalletPasswordInteractor
+  lateinit var currencyFormatUtils: CurrencyFormatUtils
```

Comment:
It's not a mandatory change but formatting values to be displayed in the fragment should be a responsibility of the presenter/interactor